cypress/support/fragments/marcAuthority/marcAuthorities.js

```diff
@@ -860,6 +860,25 @@ export default {
     });
   },
 
+  verifyHeadingTypeAccordionAccordionAndClick: () => {
```

Comment:
typo in naming
Not a useful method. If you need to open accordion to verify verifySelectedTextOfHeadingType just add click to this method